src/lang-pt_BR.js

```diff
@@ -0,0 +1,553 @@
+(function (window) {
+  /**
+   * {virtualclass1} {virtualclass2} are elements you passes with getString function eg:-
+   *   virtualclass.lang.getString('operaBrowserIssue', ['opeara', 27]);
+   *   opera and 27 will be replaced over the {virtualclass1} and {virtualclass2} respectively for particular line of 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482470)